app/controllers/reviews_controller.rb

```diff
@@ -26,9 +26,10 @@ def create
 
   def lock_controls
     unlocked = @component.rules.where(locked: false)
+    filtered_unlocked = unlocked.where(status: 'Not Yet Determined').as_json.filter { |u| u['satisfied_by'].nil? }
 
-    if unlocked.exists?(status: 'Not Yet Determined')
-      not_determined_controls = unlocked.where(status: 'Not Yet Determined').map(&:displayed_name).join(', ')
+    if filtered_unlocked.any?
+      not_determined_controls = filtered_unlocked.map { |r| "#{@component[:prefix]}-#{r['rule_id']}" }.join(', ')
```

Comment:
```suggestion
      not_determined_controls = filtered_unlocked.map { |r| "#{@component[:prefix]}-#{r['rule_id']}" }.sort.join(', ')
```

Reply:
Sort IDs for error to make it easier to read.

Reply:
used the ActiveRecord `order` method instead of ruby array `sort` for slightly better performance: https://github.com/mitre/vulcan/pull/405/commits/1c97165aee52e56a75009eae3c13a2afd91d3d45#diff-b0e95758edb5b313b1e47561c4f5cea10e74e992706d2e49503db23deae75bbeR31